osbenchmark/worker_coordinator/runner.py

```diff
@@ -1137,12 +1194,24 @@ def calculate_recall(predictions, neighbors, top_k):
                     continue
                 candidates.append(field_value)
             neighbors_dataset = params["neighbors"]
-            num_neighbors = params.get("k", 1)
-            recall_k = calculate_recall(candidates, neighbors_dataset, num_neighbors)
-            result.update({"recall@k": recall_k})
 
-            recall_1 = calculate_recall(candidates, neighbors_dataset, 1)
-            result.update({"recall@1": recall_1})
+            if "k" in params:
+                logging.getLogger(__name__).info("Calculating recall for k")
```

Comment:
Why do we need to log this? This will flood the log file

Reply:
Removed